projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-submit/case-edit-submit.component.ts

```diff
@@ -158,13 +158,13 @@ export class CaseEditSubmitComponent implements OnInit, OnDestroy {
 
   public checkYourAnswerFieldsToDisplayExists(): boolean {
      /* istanbul ignore else */
-    if (!this.eventTrigger.show_summary) {
+    if (!(this.eventTrigger.show_summary !== false)) {
```

Comment:
@DavidJayakumar - may be better to replace the double negation with `if (this.eventTrigger.show_summary === false)`. Easier to read

Reply:
In fact, isn't this the same as the original condition `!this.eventTrigger.show_summary`?

Reply:
Ah - it's because of the behaviour of `null`. OK - makes sense